src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -113,8 +113,10 @@
 
   {{ form_widget(form, vars) }}
 
-  {% if extraVars.help is defined %}
-    <small class="form-text">{{ extraVars.help }}</small>
+  {% if extraVars.help is defined or form.vars.help is defined %}
+{#    todo: which one should take the priority or both of them should be concated?#}
```

Comment:
@matks  what do you about this case?

after my changes help message can be rendered directly from type by passing an option 'help' or directly in template as we used to do. Which one has higher priority or maybe keep it how I done? :thinking:

Reply:
I can imagine the usecase "I use the same FormType 5 times in my module but I want to modify the help message in 1 of the 5 usages" so I think template help message has priority and form option comes second

Reply:
@matks done